src/map/clif.c

```diff
@@ -2386,14 +2386,26 @@ void clif_addcards2(unsigned short *cards, struct item* item) {
  * @param buf[in,out] The buffer to write to. The pointer must be valid and initialized.
  * @param item[in]    The source item.
  */
-void clif_add_random_options(unsigned char* buf, struct item* item)
+void clif_add_random_options(struct RndOptions *buf, const struct item it)
```

Comment:
Why not use pointer here?
const struct item it -> const struct item *it
And why rename it from item to it?